lua/terrortown/lang/en.lua

```diff
@@ -1248,49 +1248,49 @@ L.hilite_win_innocents = "TEAM INNOCENT WON"
 L.hilite_win_tie = "IT IS A TIE"
 L.hilite_win_time = "TIME IS UP"
 
-L.tooltip_karma_gained = "Karma gained this round:"
-L.tooltip_score_gained = "Score gained this round:"
-L.tooltip_roles_time = "Roles over time:"
+L.tooltip_karma_gained = "Karma changes for this round:"
+L.tooltip_score_gained = "Score changes for this round:"
+L.tooltip_roles_time = "Role changes for this round:"
 
 L.tooltip_finish_score_alive_teammates = "Alive teammates: {score}"
 L.tooltip_finish_score_alive_all = "Alive players: {score}"
 L.tooltip_finish_score_timelimit = "Time is up: {score}"
 L.tooltip_finish_score_dead_enemies = "Dead enemies: {score}"
 L.tooltip_kill_score = "Kill: {score}"
-L.tooltip_bodyfound_score = "Bodyfound: {score}"
+L.tooltip_bodyfound_score = "Body found: {score}"
 
 L.finish_score_alive_teammates = "Alive teammates:"
 L.finish_score_alive_all = "Alive players:"
 L.finish_score_timelimit = "Time is up:"
 L.finish_score_dead_enemies = "Dead enemies:"
 L.kill_score = "Kill:"
-L.bodyfound_score = "Bodyfound:"
+L.bodyfound_score = "Body found:"
 
 L.title_event_bodyfound = "A body was found"
-L.title_event_c4_disarm = "A C4 charge was disarmed"
-L.title_event_c4_explode = "A C4 charge exploded"
-L.title_event_c4_plant = "A C4 charge was planted"
+L.title_event_c4_disarm = "A C4 was disarmed"
+L.title_event_c4_explode = "A C4 exploded"
+L.title_event_c4_plant = "A C4 was armed"
```

Comment:
did you check this is actually true? Because you can place a C4 without arming it

Reply:
To be honest, no. Just made a logical assumption while looking at the code and related strings. Anyway, I checked it again in the game: https://www.youtube.com/watch?v=3UP6URw_twg

Reply:
video is private

Reply:
Forgot to save. Should be working now.

Reply:
Good work, nice!